segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/rocksdb/RocksDBCache.java

```diff
@@ -46,6 +47,31 @@
      */
     private static final int MAX_WRITE_AHEAD_LOG_SIZE_MB = 64;
 
+    /**
+     * Max number of in-memory write buffers (memtables) for the cache (active and immutable).
+     */
+    private static final int MAX_WRITE_BUFFER_NUMBER = 4;
+
+    /**
+     * Minimum number of in-memory write buffers (memtables) to be merged before flushing to storage.
+     */
+    private static final int MIN_WRITE_BUFFER_NUMBER_TO_MERGE = 2;
+
+    /**
+     * RocksDB allows to buffer writes in-memory (memtables) to improve write performance, thus executing an async flush
+     * process of writes to disk. This parameter bounds the maximum amount of memory devoted to absorb writes.
+     */
+    private static final int WRITE_BUFFER_SIZE_MB = 64 / MAX_WRITE_BUFFER_NUMBER;
```

Comment:
The description says that the default is 64MB per memtable but here we are splitting 64MB among 4 memtables. How did you get to the conclusion that this is appropriate? This is giving us 16MB per memtable.

Reply:
As by default `max_write_buffer_number = 2`, RocksBD is actually using at most 128MB for absorbing writes. While it can be changed, this PR provides a more conservative value for write buffering (64MB) to save up memory in scarce memory scenarios. 

Besides, when `WRITE_BUFFER_SIZE_MB` gets exposed in the configuration, the intention is to provide Pravega users with a single parameter to define the "total amount" of in-memory space for write buffering without taking care of the number of memtables, thus simplifying configuration complexity.